infra/modules/openai.bicep

```diff
@@ -34,92 +19,27 @@ resource openai 'Microsoft.CognitiveServices/accounts@2023-05-01' = {
     networkAcls: {
       defaultAction: 'Allow'
     }
-    publicNetworkAccess: publicNetworkAccess
-  }
-}
-
-resource gpt35turbodeployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = {
-  parent: openai
-  name: 'gpt-35-turbo'
-  properties: {
-    model: {
-      format: 'OpenAI'
-      name: 'gpt-35-turbo'
-      version: '0301'
-    }
-  }
-  sku: {
-    capacity: 10
-    name: 'Standard'
+    publicNetworkAccess: 'Enabled'
   }
 }
 
-// resource gpt4deployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = {
-//   parent: openai
-//   name: 'gpt-4'
-//   properties: {
-//     model: {
-//       format: 'OpenAI'
-//       name: gptModel
-//       version: gptVersion
-//     }
-//   }
-//   sku: {
-//     capacity: 10
-//     name: 'Standard'
-//   }
-// }
-
-resource adaEmbeddingsdeployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = {
+resource gptDeployment 'Microsoft.CognitiveServices/accounts/deployments@2023-05-01' = {
   parent: openai
-  name: 'text-embedding-ada-002'
+  name: 'gpt-4o'
   properties: {
     model: {
       format: 'OpenAI'
-      name: 'text-embedding-ada-002'
-      version: '2'
+      name: 'gpt-4o'
+      version: '2024-05-13'
```

Comment:
these were initially parameters on the CLI, but they had just a single option for each, so I think it was not really useful.

On the other hand if we leave this as a string parameter and the user can put anything, it can be error prone (for example set a model and use a version that is not valid).

We could add in the readme a section about using a different model/version.

what are your thoughts about that?